django_video_encoder/api.py

```diff
@@ -13,10 +13,15 @@
 from django.core import signing
 from django.core.exceptions import ObjectDoesNotExist
 from django.core.files import File
+from django.http.multipartparser import parse_header
 from django.urls import reverse
 
 from . import signals
 from .errors import ZencoderError
+from .utils import (
+    codec_width_height_to_format_label,
+    format_label_to_codec_width_height,
```

Comment:
Is `format_label_to_codec_width_height` actually used anywhere?

Reply:
völlig Recht, habe nun auch flake8 mit rein genommen. Hatte es drin und dann doch nicht so benutzt